api/src/main/java/org/openmrs/ui/framework/UiUtils.java

```diff
@@ -30,68 +21,81 @@
 import org.springframework.validation.BindingResult;
 import org.springframework.validation.Validator;
 
+import java.io.StringWriter;
+import java.text.SimpleDateFormat;
+import java.util.Collection;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Random;
+
 /**
  * Utility methods that should be available in view technologies for pages and fragments
  */
 public abstract class UiUtils {
-	
+
 	protected Locale locale;
-	
+
 	protected PageContext pageContext;
-	
+
 	protected ResourceIncluder resourceIncluder;
-	
+
 	protected FragmentIncluder fragmentIncluder;
-	
+
 	protected Formatter formatter;
-	
+
 	protected Messager messager;
-	
+
 	protected Decoratable decoratable;
-	
+
 	protected ExtensionManager extensionManager;
-	
+
 	protected ConversionService conversionService;
-	
-	public void includeCss(String file) {
-		includeCss(null, file, null);
-	}
-	
-	public void includeCss(String providerName, String file) {
-		includeCss(providerName, file, null);
-	}
-	
-	public void includeCss(String providerName, String file, Integer priority) {
-		MapResourceExtension resource = mapResource(providerName, file);
-		
-		resourceIncluder.includeResource(new Resource(Resource.CATEGORY_CSS, resource.getProviderId(), "styles/"
-		        + resource.getResourceId(), priority));
-	}
-	
-	public void includeJavascript(String file) {
-		includeJavascript(null, file, null);
-	}
-	
-	public void includeJavascript(String providerName, String file) {
-		includeJavascript(providerName, file, null);
-	}
-	
-	public void includeJavascript(String providerName, String file, Integer priority) {
-		MapResourceExtension resource = mapResource(providerName, file);
-		
-		resourceIncluder.includeResource(new Resource(Resource.CATEGORY_JS, resource.getProviderId(), "scripts/"
-		        + resource.getResourceId(), priority));
-	}
-	
+
+    public void includeCss(String file) {
+        includeCss(null, file, null);
+    }
+
+    public void includeCss(String providerName, String file) {
+        includeCss(providerName, file, null);
+    }
+
+    public void includeCss(String providerName, String file, Integer priority) { includeCss(providerName, file, priority, false);}
+
+    public void includeCss(String providerName, String file, Integer priority, Boolean excludeStylesPrefixFromPath) {

```

Comment:
I'm not crazy about the way I chose to do this (adding an "excludeBy..." flag, but I couldnt' come up with a better way.

Reply:
`pathIsRelativeToStyles`, defaulting to true?

Reply:
Also, any reason not to use the primitive `boolean`?

Reply:
I like that better, will switch, thanks Brandon!